File: db4oj/core/src/com/db4o/foundation/BlockingQueueStoppedException.java
/* Copyright (C) 2007  Versant Inc.  http://www.db4o.com */

package com.db4o.foundation;

import com.db4o.*;


/**
 * @exclude
 */
public class BlockingQueueStoppedException extends RuntimeException {
	
	public BlockingQueueStoppedException(){
		super();
		if(DTrace.enabled){
			DTrace.BLOCKING_QUEUE_STOPPED_EXCEPTION.log();
		}
	}

}

File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Foundation/BlockingQueueStoppedException.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using System;
using Db4objects.Db4o;

namespace Db4objects.Db4o.Foundation
{
	/// <exclude></exclude>
	[System.Serializable]
	public class BlockingQueueStoppedException : Exception
	{
		public BlockingQueueStoppedException() : base()
		{
			if (DTrace.enabled)
			{
				DTrace.BlockingQueueStoppedException.Log();
			}
		}
	}
}
